docs/rageagainstthevm.md

```diff
@@ -0,0 +1,3 @@
+- **Name** George
+- **Occupation** Managing Partner for Digital Forge Technologies
```

Comment:
Looks like you may have forgotten an email address.  It's :ok: to put something fake in though since this is a public repository.  :smiley: